src/shared/ECS/Components.hpp

```diff
@@ -0,0 +1,57 @@
+/*
+** EPITECH PROJECT, 2022
+** Armor.hpp
+** File description:
+** .
+*/
+
+#pragma once
+
+#include "ECS.hpp"
+#include "Manager.hpp"
+enum ComponentType { ARMOR, HEALTH, POSITION };
+
+namespace Armor {
+    struct data {
```

Comment:
Please call this truct "Comp" or "Component" so it reflects the fact it remains the main representation of the component.

Reply:
sure thing